### DISCUSSION TITLE
Regarding student identity verification

### DISCUSSION THREAD

**Discussion by JarrettJ05** [community/community#136178] [GitHub Education]
### Hi everyone,

Why can't I use photo files? My devices, whether computers or mobile phones, seem to only be able to use the front camera, which can't take clear photos at all. And GitHub has also closed manual review, I am confused about GitHub's operation

**Comment by AnshulJagotra** [comment]
Hi JarrettJ05,

It seems you're facing issues with student identity verification because your devices are only allowing you to use the front camera, which isn't providing clear photos. Unfortunately, GitHub requires a clear and readable photo of your student ID, and since they've closed manual review, there's no alternative method available right now.

Here are a few things you can try:

Use a different device: If possible, borrow a device with a better rear camera to take a clearer picture.
Check camera settings: Sometimes, adjusting the camera settings (like resolution) can improve the photo quality.
Good lighting: Ensure you're in a well-lit environment to improve the clarity of the photo.
Try a different browser or app: Occasionally, using a different browser or updating the current one can resolve camera issues.
If none of these work, you might want to reach out to GitHub Support for further assistance, although options might be limited without manual review.
